dcr/templates/setup-vm-and-execute-tests.yml

```diff
@@ -0,0 +1,211 @@
+parameters:
+  - name: scenarios
+    type: object
+
+  - name: distros
+    type: object
+
+  - name: rgPrefix
+    type: string
+
+jobs:
+  - job: "CreateVM"
+    displayName: "Setup VM and Run Test"
+
+    strategy:
+      matrix:
+        ${{ each distro in parameters.distros }}:
+          ${{ each scenario in parameters.scenarios }}:
```

Comment:
Looks to me it's gonna run all scenarios for each distro but how can I run cases like distro1 for scenario1, distro2 for scenario2 not all?